### ISSUE TITLE
Varimathras is not present in Undercity

### ISSUE THREAD

**Issue by konopkap** [chromiecraft/chromiecraft#5844]
### What client do you play on?

enUS

### Faction

Horde

### Content Phase:

70

### Current Behaviour

Multiple quests in Silverpine Forest (i.e. [Prove Your Worth](https://www.wowhead.com/wotlk/quest=421/prove-your-worth)) references [Varimathras](https://www.wowhead.com/wotlk/npc=2425/varimathras) who seems (from quest description) to be under [Sylvanas](https://www.wowhead.com/wotlk/npc=10181/lady-sylvanas-windrunner) command. 

However he's not in the Undercity. I suppose it's because CC is running on 3.3.5a and [Varimathras](https://www.wowhead.com/wotlk/npc=2425/varimathras) has disapeared in Wrathgate quest chain.

### Expected Blizzlike Behaviour

Varimathras is present in Undercity as we're currently in 70 bracket.

### Source

https://www.wowhead.com/wotlk/npc=2425/varimathras - NPC description
https://wowpedia.fandom.com/wiki/Varimathras - presence description

### Steps to reproduce the problem

1. Go to 56.6 90.8
2. See that Varimathras is not there

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/fb7164be05ff507a0e25200c9e267b87186e2850

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-bg-item-reward](https://github.com/azerothcore/mod-bg-item-reward)
- [mod-BreakingNewsOverride](https://github.com/AnchyDev/BreakingNewsOverride)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chat-transmitter](https://github.com/azerothcore/mod-chat-transmitter)
- [mod-cta-switch](https://github.com/azerothcore/mod-cta-switch)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna](https://github.com/azerothcore/mod-eluna)
- [mod-instanced-worldbosses](https://github.com/nyeriah/mod-instanced-worldbosses)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-low-level-rbg](https://github.com/azerothcore/mod-low-level-rbg)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-rdf-expansion](https://github.com/azerothcore/mod-rdf-expansion)
- [mod-transmog](https://github.com/azerothcore/mod-transmog)
- [mod-weekend-xp](https://github.com/azerothcore/mod-weekend-xp)
- [mod-zone-difficulty](https://github.com/azerothcore/mod-zone-difficulty)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-event-scripts](https://github.com/55Honey/Acore_eventScripts)
- [lua-exchange-npc](https://github.com/55Honey/Acore_ExchangeNpc)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by konopkap** [comment]
Wanted to add that there are other NPC's which are behave like Wrathgate chain quest has been completed. I.e. [Overseer Kraggosh](https://www.wowhead.com/wotlk/npc=36217/overseer-kraggosh) is present in the Apothecarium  Or [Master Apothecary Faranell](https://www.wowhead.com/classic/npc=2055/master-apothecary-faranell) is talking about beeing overseeing by his guests (which happened after the Wrathgate).

IMO it looks like whole Undercity is in the state after Battle for Undercity (bot don't know how to check\prove that). Should I create separate ticket for that?

**Comment by Gultask** [comment]
No, it's fine. There really isn't a pre-Battle for Undercity stage, but we *did* change the guards from Kor'kron to the guardians.
We didn't add Varimathras back *yet* for a couple of reasons, the main one is that it *could* break quests or RP in the Keep.

**Comment by Gultask** [comment]
@heyitsbench Could you link the issue in mod-prog? 🥺

**Comment by konopkap** [comment]
> 

In what Keep exactly?

**Comment by heyitsbench** [comment]
Thanks for the report! ❤️
 
> In what Keep exactly?

I believe Gultask is/was worried some RP would look odd or break throughout the Undercity.
